### ISSUE TITLE
Add seed data script (mock data generation)

### ISSUE THREAD

**Issue by adeebahmed** [target/consensource#18]
It'd be handy to have a toggleable script that populates Consensource with mock data on start. This script would use the CLI to sign mock transactions on to the underlying blockchain. 

Acceptance Criteria:
- [ ] Uses CLI to generate transactions
- [ ] 5 factories
- [ ] 2 standards
- [ ] 2 auditors
- [ ] 3 certifications granted to factories

**Comment by crawfordleeds** [comment]
Hey, I'd like to help with this. Had a thought and question I wanted to run by you if you thought this was on the right track.

First, the question. Isn't this already partially complete with the `creg genesis -g example-genesis.yaml` command? In other words, isn't it just a matter of changing the default example-genesis.yaml file to meet the acceptance criteria?

If not, then I have an idea to run by you before I go down the wrong path.

Wouldn't it just be a matter of adding a conditional to the entry point of the cli in docker-compose.yaml?

From the lines...
https://github.com/target/consensource/blob/b078c406752313400a31f9565c302d8bce8193ad/docker-compose.yaml#L28-L36

We could add something like:

```
if [${some_arg_init_toggle}]; then
script_meeting_acceptance_criteria_here
fi;
```

Then we could toggle this initialization with something by running `some_arg_init_toggle docker-compose -f docker-compose.yaml up`.

I'm not sure about all the syntax here, I'd have to re-familiarize myself but wanted to first get your feedback on this path.

Thanks!

**Comment by adeebahmed** [comment]
Sorry for the tardy response. 

This approach is fine and your proposed solution would seed the system with mock data at genesis.  To simulate the transactional nature of these interactions between entities it'd be best to have this as a script that runs post genesis.

https://github.com/target/consensource/blob/b078c406752313400a31f9565c302d8bce8193ad/docker-compose.yaml#L54-L59

The sawadm genesis subcommand produces a file for use during the initialization of a validator. A network requires an initial block (known as the genesis block) whose signature will determine the block chain id. This initial block is produced from a list of batches, which will be applied at genesis time.

There seem to be some Postgres connectivity issues during startup (at runtime) that is preventing the validator ingest of those batches that get created at startup (from the docker-compose.yml) i've opened an issue for this body of work: https://github.com/target/consensource/issues/26

**Comment by adeebahmed** [comment]
This is the proper issue to resolve seeding data at genesis time: https://github.com/target/consensource/issues/34

**Comment by Patrick-Erichsen** [comment]
This issue was closed by #41 - the problem was fixed by bumping our Sawtooth version to 1.2.
